pkgs/by-name/ti/timg/package.nix

```
@@ -20,13 +20,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "timg";
-  version = "1.6.1";
+  version = "1.6.2";
 
   src = fetchFromGitHub {
     owner = "hzeller";
     repo = "timg";
     rev = "v${finalAttrs.version}";
```

It would be nice to update this, if it's ok for you:

```suggestion
    tag = "v${finalAttrs.version}";
```

---

sounds good. I didn't know there was a way to express that `tag`